src/resources/views/crud/reorder.blade.php

```diff
@@ -221,8 +221,8 @@ function tree_element($entry, $key, $all_entries, $crud)
 @endsection
 
 @section('after_scripts')
-    <script src="{{ asset('packages/jquery-ui-dist/jquery-ui.min.js') }}" type="text/javascript" ></script>
-    <script src="{{ asset('packages/nestedSortable/jquery.mjs.nestedSortable2.js') }}" type="text/javascript" ></script>
+    @basset('https://unpkg.com/jquery-ui@1.13.2/dist/jquery-ui.min.js')
+    @basset('https://cdnjs.cloudflare.com/ajax/libs/nestedSortable/2.0.0/jquery.mjs.nestedSortable.min.js')
```

Comment:
This isn't working for me. Once I drag something:

![CleanShot 2023-01-03 at 22 04 10](https://user-images.githubusercontent.com/1032474/210432773-c86ad52e-a8c5-4821-bc9f-cb9975425197.png)

Reply:
Fixed in https://github.com/Laravel-Backpack/CRUD/pull/4930